### PR TITLE
allow comment in between tokens of a field definition ( #131 )

### PR DESCRIPTION
**Pull request by emicklei**

### CODE REVIEW

File: comment_test.go [hunk 1]
```diff
@@ -310,3 +310,26 @@ message Foo {
 		t.Fatalf("got [%v] want [%v]", got, want)
 	}
 }
+
+func TestNormalFieldInlineComment(t *testing.T) {
+	src := `message Example {
+		/* i'm */ optional /* a comment */ bool /* too */ field /* hard */ = /* to */ 1 /* read */;
```
Comment (zack-littke-smith) [inline comment_test.go:316]: Thank you, difficult to read indeed :smile:

File: field.go [hunk 2]
```diff
@@ -90,24 +97,53 @@ done:
...
+			if tok == tNUMBER {
+				// put it back so we can use the generic nextInteger
+				p.nextPut(pos, tok, lit)
+				i, err := p.nextInteger()
+				if err != nil {
+					return p.unexpected(lit, expected, f)
+				}
+				f.Sequence = i
+				expectedToken = -1
+				continue
```
Comment (zack-littke-smith) [inline field.go:141]: `break`?

File: field.go [hunk 3]
```diff
@@ -90,24 +97,53 @@ done:
...
+			if f.InlineComment == nil {
+				f.InlineComment = c
+			} else {
+				f.InlineComment.Merge(c)
+			}
+			continue
+		}
+		if tok != expectedToken {
+			return p.unexpected(lit, expected, f)
+		} else {
```
Comment (zack-littke-smith) [inline field.go:116]: `else` is implied here since the above branch would return, if you wanted to flatten the rest of this loop.

**Review by zack-littke-smith — APPROVED** [review]
Not that you needed my review, thanks a bunch for this

File: field.go (hunk 3 above)
Comment (emicklei) [reply field.go:116]: thx for your comments, will process them now

**Review by zack-littke-smith — APPROVED** [review]
@emicklei not sure if you're awaiting a followup review, but this looks good to me!

**Pull request merged**